packages/search/src/features/registration/deduplicate/deduplicate.test.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import { startContainer, stopContainer } from './deduplicate'
+import { indexComposition } from '@search/elasticsearch/dbhelper'
+import * as elasticsearch from '@elastic/elasticsearch'
+import { searchForDuplicates } from './service'
+import { StartedElasticsearchContainer } from 'testcontainers'
+
+jest.setTimeout(10 * 60 * 1000)
+
+let container: StartedElasticsearchContainer
+let client: elasticsearch.Client
+
+beforeAll(async () => {
+  container = await startContainer()
+  const host = container.getHost()
+  const port = container.getMappedPort(9200)
+
+  client = new elasticsearch.Client({
+    node: `http://${host}:${port}`
+  })
+})
+
+describe('Deduplication tests', () => {
+  afterAll(async () => {
+    try {
+      await client.close()
+    } catch (error) {
+    } finally {
+      await stopContainer(container)
+    }
+  })
```

Comment:
Would it make sense to move this block to the root level of the file, next to beforeAll?

Reply:
`beforeAll(async () => {
  container = await startContainer()
  const host = container.getHost()
  const port = container.getMappedPort(9200)

  client = new elasticsearch.Client({
    node: `http://${host}:${port}`
  })
})

afterAll(async () => {
  try {
    await client.close()
  } catch (error) {
  } finally {
    await stopContainer(container)
  }
})`

in this mannere?

Reply:
Precisely 👍